### PR TITLE
Remote link changes

### PR DESCRIPTION
**Pull request by feldergast**
Changed how events get tagged with remote link information to better handle parallel loading.  Also add a single file mode to parallel loading.

### CODE REVIEW

File: src/sst/core/link.cc [hunk 1]
```diff
@@ -87,6 +100,10 @@ Link::prepareForComplete()
     if ( POLL == type ) { delete pair_link->send_queue; }
 
     pair_link->send_queue = nullptr;
+
+    // If my pair link is a SYNC link, also need to call
+    // finalizeConfiguration on it
```
Comment (jpkenny) [inline src/sst/core/link.cc:105]: prepareForComplete

Comment (feldergast) [reply src/sst/core/link.cc:105]: I have another PR that's waiting on this one to merge so I can rebase.  If you don't find anything substantial in this PR, I've made the edit in the new one.

File: src/sst/core/sync/threadSyncDirectSkip.cc [hunk 2]
```diff
@@ -54,11 +54,11 @@ ThreadSyncDirectSkip::~ThreadSyncDirectSkip()
             CALL_INFO, 1, 0, "ThreadSyncDirectSkip total wait time: %lg seconds.\n", totalWaitTime);
 }
 
-ActivityQueue*
-ThreadSyncDirectSkip::getQueueForThread(int UNUSED(tid))
-{
-    return nullptr;
-}
+// ActivityQueue*
```
Comment (jpkenny) [inline src/sst/core/sync/threadSyncDirectSkip.cc:57]: Can delete this commented out function? Declaration is deleted from header.

Comment (feldergast) [reply src/sst/core/sync/threadSyncDirectSkip.cc:57]: Yep, I usually catch those while review the diffs.

**Review by jpkenny — APPROVED** [review]
Ok to resolve 2 minor comments in next PR. Everything else LGTM so approving.

**Pull request merged**